core/src/main/scala/kafka/api/ApiVersion.scala

```diff
@@ -76,7 +76,9 @@ object ApiVersion {
     // Introduced DeleteGroupsRequest V0 via KIP-229, plus KIP-227 incremental fetch requests,
     // and KafkaStorageException for fetch requests.
     "1.1-IV0" -> KAFKA_1_1_IV0,
-    "1.1" -> KAFKA_1_1_IV0
+    "1.1" -> KAFKA_1_1_IV1,
+    // Bumped the version of ApiVersionsRequest/Response for KIP-219.
```

Comment:
In order to be consistent with other comments, it may be better to say `Introduced ApiVersionsRequest V2 via KIP-219`. Also, can you move `"1.1" -> KAFKA_1_1_IV1` to the last entry?

Reply:
Done